### DISCUSSION TITLE
Modifying views

### DISCUSSION THREAD

**Discussion by gl2007** [codeigniter4/shield#817] [Q&A] [answered]
How can I modify the views provided by Shield viz. login, register, (and reset password)? Basically, I want to customize the UI, CSS etc, and not change any functionality.

Haven't got email to work yet but will create that as another question.

Sounds like I have to copy out the views from source and customize them?
For e.g. login form:
https://github.com/codeigniter4/shield/blob/develop/src/Views/login.php

**Comment by datamweb** [comment, marked as answer]
see #470
